### ISSUE TITLE
Add brew to the path on linux

### ISSUE THREAD

**Issue by ajr-dev** [nicknisi/dotfiles#153]
It's a step that needs to be added if brew is going to be installed on linux.
```
# Add Homebrew to your PATH and to your bash shell profile script
# either ~/.profile on Debian/Ubuntu or ~/.bash_profile on CentOS/Fedora/RedHat
test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)
test -d /home/linuxbrew/.linuxbrew && eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
test -r ~/.bash_profile && echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.bash_profile
echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.profile
```
It's only useful until zsh is the main shell. But it's needed to run `brew bundle`.

**Comment by nicknisi** [comment]
Good call! I think this should be resolved in #147. I've been meaning to get this merged and need to add a few more tweaks.

**Comment by ajr-dev** [comment]
Actually to have it in the path is enough:
```
# Add Homebrew to your PATH
test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)
test -d /home/linuxbrew/.linuxbrew && eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
```

**Comment by nicknisi** [comment]
I believe this should be fixed now that #147 is merged!
